src/components/OurValues/data.js

```diff
@@ -0,0 +1,22 @@
+import inclusivityImg from '../../../src/images/22-web_essential-512.png'
+import communityImg from '../../../src/images/iconfinder_05-web_essential_3401825.svg'
+import developmentImg from '../../../src/images/iconfinder_27-web_essential_3401845.svg'
+
+const valuesData = [
+  { src: inclusivityImg,
+    headline: "Inclusivity",
+    desc: "We are an inclusive tech community that cares for our member's professional development" },
+  { src: communityImg,
+    headline: "Community",
+    desc: "We are an inclusive tech community that cares for our member's professional development" },
+  { src: developmentImg,
+    headline: "Development",
```

Comment:
I noticed that the descriptions are the same: desc: "We are an inclusive tech community that cares for our member's professional development"

But I think the figma has three different descriptions

Reply:
Oh yeah, thanks Jen for catching that !